rcgen/git/utils.js

```diff
@@ -0,0 +1,15 @@
+// @ts-check
+
+const {gitIgnoreFile} = require('./files');
+const {merge} = require('@rcgen/patchers');
+
+/**
+ * @param {...string} filenames
+ *
+ * @returns {import('@rcgen/core').Patcher<string[]>}
+ */
+function ignoreFilesWithGit(...filenames) {
```

Comment:
what is the reasoning for naming those functions `ignoreFilesWithFoo`? Intuitively I would name those `fooignoreFiles`...

Reply:
Ok i will change this to `gitIgnoreFiles`